pkg/scheduler/framework/runtime/framework.go

```diff
@@ -726,10 +726,14 @@ func (f *frameworkImpl) RunPreFilterPlugins(ctx context.Context, state *framewor
 			}
```

Comment:
do we need to record unschedulable plugins in the branches above - lines 715 and 720? or only when the plugin filters out some nodes (as title of PR indicates)

Reply:
The previous implementation that unschedulable plugins was also added in lines 713 and 714

Reply:
I think I understand now. We [set the plugin name](https://github.com/kubernetes/kubernetes/blob/b498eb97406f780fbbeb39204f50bc473e64af56/pkg/scheduler/framework/runtime/framework.go#L714), and later we add it via [this call](https://github.com/kubernetes/kubernetes/blob/b498eb97406f780fbbeb39204f50bc473e64af56/pkg/scheduler/schedule_one.go#L479)?

But if more than one plugin returns Unschedulable (or that, +a plugin returns UnschedulableAndUnresolvable), we may miss some plugins, right?